addons/mail/static/src/utils/common/hooks.js

```diff
@@ -210,6 +210,9 @@ export function useVisible(refName, cb, { ready = true } = {}) {
                     observer.unobserve(el);
                 };
             }
+            if (ready) {
```

Comment:
I think this should just be `else`, not ready can also lead to undefined, this is the initial state too.

Reply:
Shouldn't the return of the `if` simply be `setValue(undefined)` as it is the cleanup function for when dependencies are changing?

Reply:
done